e2e/lib/k8up.bash

```diff
@@ -74,11 +97,16 @@ apply() {
 	kubectl apply -f "debug/${1}/main.yml"
 }
 
-given_a_subject() {
-	kubectl delete namespace k8up-e2e-subject --ignore-not-found
+given_a_clean_ns() {
+	kubectl delete namespace "${DETIK_CLIENT_NAMESPACE}" --ignore-not-found
 	kubectl delete pv subject-pv --ignore-not-found
-	kubectl create namespace k8up-e2e-subject || true
+	kubectl create namespace "${DETIK_CLIENT_NAMESPACE}" || true
```

Comment:
Probably a more reliable command, in terms of exit code.
```suggestion
	kubectl create namespace --save-config --dry-run=client -o yaml "${DETIK_CLIENT_NAMESPACE}" | kubectl apply -f -
```

Reply:
Actually, if we delete the namespace before, why is there a `|| true` ?

Reply:
I think so as well.